azure-pipelines.yml

```diff
@@ -15,6 +15,8 @@ variables:
   - group: docker-settings
   - name: app_name
     value: pr-$(System.PullRequest.PullRequestNumber)
+  - name: TF_VAR_PR_NUMBER
```

Comment:
Terraform variables are usually lowercase, so here we should have `TF_VAR_pr_number`

Reply:
I used capitals according to this doc  : 

"Then, in your Terraform configuration, you have to define the variables with capital letters.

Why? Because Azure DevOps will always transform pipeline variables to uppercase environment variables."